templates/about.html

```diff
@@ -0,0 +1,365 @@
+             Data Science/Predictive Analytics end of the data spectrum.</p>
+          <ul role="list" class="mt-6 flex gap-x-6">
+            <li>
+              <a href="https://github.com/nathanaelrobinson" class="text-gray-400 hover:text-gray-500">
+                <span class="sr-only">Github</span>
+                <svg class="size-6" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true">
+                  <path fill-rule="evenodd" d="M12 2C6.477 2 2 6.484 2 12.017c0 4.425 2.865 8.18 6.839 9.504.5.092.682-.217.682-.483 0-.237-.008-.868-.013-1.703-2.782.605-3.369-1.343-3.369-1.343-.454-1.158-1.11-1.466-1.11-1.466-.908-.62.069-.608.069-.608 1.003.07 1.531 1.032 1.531 1.032.892 1.53 2.341 1.088 2.91.832.092-.647.35-1.088.636-1.338-2.22-.253-4.555-1.113-4.555-4.951 0-1.093.39-1.988 1.029-2.688-.103-.253-.446-1.272.098-2.65 0 0 .84-.27 2.75 1.026A9.564 9.564 0 0112 6.844c.85.004 1.705.115 2.504.337 1.909-1.296 2.747-1.027 2.747-1.027.546 1.379.202 2.398.1 2.651.64.7 1.028 1.595 1.028 2.688 0 3.848-2.339 4.695-4.566 4.943.359.309.678.92.678 1.855 0 1.338-.012 2.419-.012 2.747 0 .268.18.58.688.482A10.019 10.019 0 0022 12.017C22 6.484 17.522 2 12 2z" clip-rule="evenodd" />
+                </svg>
+              </a>
+            </li>
+            <li>
+              <a href="https://github.com/nathanaelrobinson" class="text-gray-400 hover:text-gray-500">
+                <span class="sr-only">LinkedIn</span>
+                <svg class="size-6" fill="currentColor" viewBox="0 0 20 20" aria-hidden="true">
+                  <path fill-rule="evenodd" d="M16.338 16.338H13.67V12.16c0-.995-.017-2.277-1.387-2.277-1.39 0-1.601 1.086-1.601 2.207v4.248H8.014v-8.59h2.559v1.174h.037c.356-.675 1.227-1.387 2.526-1.387 2.703 0 3.203 1.778 3.203 4.092v4.711zM5.005 6.575a1.548 1.548 0 11-.003-3.096 1.548 1.548 0 01.003 3.096zm-1.337 9.763H6.34v-8.59H3.667v8.59zM17.668 1H2.328C1.595 1 1 1.581 1 2.298v15.403C1 18.418 1.595 19 2.328 19h15.34c.734 0 1.332-.582 1.332-1.299V2.298C19 1.581 18.402 1 17.668 1z" clip-rule="evenodd" />
+                </svg>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </li>
+      <li class="flex flex-col gap-10 pt-12 sm:flex-row">
+        <img class="aspect-[4/5] w-52 flex-none rounded-2xl object-cover" src="/assets/img/kevin-profile.jpeg" alt="Kevin Koenitzer">
+        <div class="max-w-xl flex-auto">
+          <h3 class="text-lg/8 font-semibold tracking-tight text-gray-900">Kevin Koenitzer</h3>
+          <p class="text-base/7 text-gray-600 italic">Managing Partner</p>
+          <span class="inline-flex items-center rounded-full bg-light-green px-2 py-1 text-xs font-medium text-gray-dark">Product Analytics</span>
+          <span class="inline-flex items-center rounded-full bg-teal px-2 py-1 text-xs font-medium text-gray-light">Business Intelligence</span>
+          <span class="inline-flex items-center rounded-full bg-pink px-2 py-1 text-xs font-medium text-gray-light">Leadership</span>
```

Comment:
```suggestion
          <span class="inline-flex items-center rounded-full bg-pink px-2 py-1 text-xs font-medium text-gray-light">Management</span>
```
feel like "leadership" is a little too broad and less clear about implying "running a business department". 
Thoughts?

Reply:
I like it!